debian/changelog

```diff
@@ -1,3 +1,16 @@
+daos (2.3.100-19) unstable; urgency=medium
+  [ Wang Shilong ]
+  * Add daos_debug_set_params to daos-client-tests rpm for fault injection test.
+
+ -- Wang Shilong <shilong.wang@intel.com>  Mon, 8 Aug 2022 10:48:00 -0400
+
+daos (2.3.100-18) unstable; urgency=medium
+  [Michael MacDonald]
+  * Bump min supported go version to 1.16
+
+ -- Michael MacDonald <mjmac.macdonald@intel.com> Tue, Jul 26 2022 10:10:00
+-0400
```

Comment:
Bad merge?

Reply:
No, i was aware  this is missed, so this is new part that i added.